transaction/core/src/ring_signature/generator_cache.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! A simple generator cache
+
+use super::{generators, PedersenGens};
+use crate::TokenId;
+use alloc::collections::BTreeMap;
+
+/// GeneratorCache is a simple object which caches computations of
+/// generator: TokenId -> PedersenGens
+///
+/// This is intended just to be used in the scope of a single transaction,
```

Comment:
In the scope of constructing or verifying a single transaction?

Reply:
Yea that's how it's used in this commit